### PR TITLE
Riyuan: Separated Project and contact to a new page

### PR DESCRIPTION
**Pull request by Riyuanliu**
Created a route file that move Project and contact page to a new page.

L=ENG-153
image

### CODE REVIEW

File: js/src/personal/riyuan/components/project/Project.tsx [hunk 1]
```diff
@@ -1,12 +1,9 @@
-import { Text, Title, Container } from '@mantine/core'
+import { Container, Title, Text } from '@mantine/core'
 import styles from '@/personal/riyuan/Riyuan.module.css'
 
-export function Project() {
+export function RiyuanProjectsPage() {
```
Comment (arklian) [inline js/src/personal/riyuan/components/project/Project.tsx:4]: File name should match component name.
We may want to standardize on RiyuanProjects.page.tsx though, to denote top level react components.

Comment (arklian) [inline js/src/personal/riyuan/components/project/Project.tsx:4]: Also you should comment above the component what the high level purpose of the component is.

Comment (Riyuanliu) [reply js/src/personal/riyuan/components/project/Project.tsx:4]: done

File: js/src/personal/riyuan/components/project/RiyuanProjectsPage.tsx [hunk 2]
```diff
@@ -1,12 +1,10 @@
-import { Text, Title, Container } from '@mantine/core'
+import { Container, Title, Text } from '@mantine/core'
 import styles from '@/personal/riyuan/Riyuan.module.css'
 
-export function Project() {
+// This component renders the projects page
```
Comment (arklian) [inline js/src/personal/riyuan/components/project/RiyuanProjectsPage.tsx:4]: nit: Should use the JSDoc format a bit for it: `/** Example Comment */`
Also probably want to be as concise as possible while also giving more context. 
Similar to commit message summaries, the comments should be written in an imperative mood.

Focus on the Why and not the What. 

// This component renders the projects page ->
/** Renders a Page that highlights different projects completed by me for visitors to get a sense of the work I've completed */

**Comment by Riyuanliu** [comment]
Change-Id  was closed at https://gerrithub.io/q/

**Pull request closed without merging**